templates/etc/fastd/fastd.conf.erb

```diff
@@ -17,7 +17,7 @@ on up "
  ip link set address <%= @vpn_mac %> dev $INTERFACE
  /usr/sbin/batctl -m bat-<%= @mesh_code %> if add $INTERFACE
  ip link set address <%= @mesh_mac %> dev bat-<%= @mesh_code %>
- ifup bat-<%= @mesh_code %>
+ ip link set up dev bat-<%= @mesh_code %>
```

Comment:
``ifup`` does more than just bring the device up, it also configures the batman interface and put it into the mesh bridge. The problem you hit into is caused by debian is remembering the configurion it did and so refuse to ifup the device again. I propose the use ``ifup --force bat-<% @mesh_code %>`` instead.